### PR TITLE
[rel-5_0] Ported back translatable messages from master to rel-5_0

### PR DESCRIPTION
**Pull request by urbalazs**
Hi @mgruner 
I reviewed the Kernel/Modules/*.pm files in rel-5_0, and I ported back the translatable strings from master. I also ported back this commit: https://github.com/OTRS/otrs/commit/99f85458b8d9011b808c25b0ebebd655ea820ba2

### CODE REVIEW

File: Kernel/Output/HTML/Templates/Standard/CustomerLogin.tt [hunk 1]
```diff
@@ -115,8 +115,8 @@
                     <input type="hidden" name="Action" value="CustomerCreateAccount"/>
                     <input type="hidden" name="Lang" value="[% Env("UserLanguage") | html %]"/>
                     <div>
-                        <label for="Title">[% Translate("Title") | html %]</label>
-                        <input title="[% Translate("How we should address you") | html %]" type="text" name="Title"  id="Title" maxlength="50" value="[% Data.UserTitle | html %]"/>
+                        <label for="Title">[% Translate("Title or salutation") | html %]</label>
```
Comment (mgruner) [inline Kernel/Output/HTML/Templates/Standard/CustomerLogin.tt]: These seem to be unrelated changes that should go into another PR or bugfix.

File: var/httpd/htdocs/skins/Customer/default/css/Core.Login.css [hunk 2]
```diff
@@ -178,10 +178,7 @@
  * @subsection  Custom Field Sizes
  */
 
-#Signup #Title {
-    width: 55px;
-}
-
+#Signup #Title,
```
Comment (mgruner) [inline var/httpd/htdocs/skins/Customer/default/css/Core.Login.css]: This is also about a different matter.

**Comment by mgruner** [comment]
Hello @urbalazs. This PR contains unrelated changes, please check. Besides that, I would like to ask you to provide the changes for rel-5_0 in smaller chunks of a few files each for better testing/review possibilities. I already burnt my fingers with cherrypicking such changes from master...

I'll close this huge PR for now. Looking forward to more PRs from you, we'll make sure to get your changes into rel-5_0!

**Comment by urbalazs** [comment]
Hi @mgruner,
Thank you for the suggestion, next time I will make smaller PRs. I removed from this one the backporting of 'Title or salutation' issue from these files: AdminUser.tt, CustomerLogin.tt, Core.Login.css.
May I reopen this huge PR (last time), or should I split it into smaller ones? This PR contains only modifications, which are currently in master.

**Comment by mgruner** [comment]
I would be grateful if you could create a handful of smaller PRs from this, please.
